docs/docs/sourcing-from-wordpress.md

```diff
@@ -9,6 +9,8 @@ WordPress is a free and open-source content management system (CMS). Let's say y
 _Note: this guide uses the `gatsby-starter-default` to provide you with the knowledge necessary to start working with WordPress but if you get stuck at some point of the guide feel free to use
 [this example](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-wordpress) to gain extra insights._
 
+_Note: If you are using Gastby v2 then use [`gatsby-starter-wordpress`](https://github.com/GatsbyCentral/gatsby-starter-wordpress) instead, follow the instructions [here](https://github.com/GatsbyCentral/gatsby-starter-wordpress)._
```

Comment:
Why would linking to this be needed? Our `/examples/using-wordpress` should also work with Gatsby v2.

Reply:
https://github.com/gatsbyjs/gatsby/issues/6231#issuecomment-401463571

Reply:
I personally think it would be better to update the example hosted in the Gatsby repo than to link elsewhere. What about that example on GatsbyCentral is necessary? Everything we recommend should be usable with Gatsby v2 at this point.